orders/models.py

```diff
@@ -71,3 +71,24 @@ def cancel_order(self):
 
     def get_order_total_price(self):
         return self.price_per_day * (self.end_date - self.start_date).days
+
+    def get_order_status(self):
+        if self.status == StatusOptions.Pending:
+            return "Pending"
+        elif self.status == StatusOptions.Approved:
+            return "Approved"
+        elif self.status == StatusOptions.Canceled:
+            return "Canceled"
+        elif self.status == StatusOptions.OnGoing:
+            return "On going"
+        return "Finished"
+
+    def is_the_order_cancelable(self):
+        if self.status != StatusOptions.OnGoing or self.status != StatusOptions.Finished:
+            return True
```

Comment:
I think you should also return true in the status `Pending`  if the user wants to cancel the order.

Reply:
The option of cancelling the order is when the order is not finished or currently happening.
The pending status is included in that scenario.